octis/preprocessing/preprocessing.py

```diff
@@ -1,35 +1,66 @@
-import string
-from typing import List, Union
+# mypy: ignore-errors
```

Comment:
Are these still necessary in the OCTIS repo?

Reply:
The code in the original OCTIS repo doesn't strictly follow mypy or flake8. I think we could remove it if we are not going to enforce these requirements.